pkgs/by-name/cu/cursewords/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cursewords";
+  version = "1.1";
+
+  src = fetchPypi {
```

Comment:
in general for python packages fetching from github is preferred, because published packages sometimes don't include tests. I'll let you to choose whats more appropriate

Reply:
Thanks for the hint! Switched to github.